samples/native-lib/README.md

```diff
@@ -45,11 +45,20 @@ will be generated.
 
 ## Run workload
 
+### Linux
+
 ```bash
 cd build
 ./iwasm --native-lib=libtest_add.so --native-lib=libtest_sqrt.so wasm-app/test.wasm
 ```
 
+### macOS
+
+```bash
+cd build
+./iwasm --native-lib=libtest_add.dylib --native-lib=libtest_sqrt.dylib wasm-app/test.wasm
```

Comment:
PR #1546 changes L52 as:
`./iwasm --native-lib=./libtest_add.so --native-lib=./libtest_sqrt.so wasm-app/test.wasm`
Shall we change this line to:
`./iwasm --native-lib=./libtest_add.dylib --native-lib=./libtest_sqrt.dylib wasm-app/test.wasm`

Reply:
i'm sure this works without `./` here on my env. (x86-64 macOS)
```
spacetanuki% ./iwasm --native-lib=libtest_add.dylib --native-lib=libtest_sqrt.dylib wasm-app/test.wasm
Hello World!
10 + 20 = 30
sqrt(10, 20) = 500
spacetanuki% 
```